mozjs-sys/Cargo.toml

```diff
@@ -2,16 +2,24 @@
 name = "mozjs_sys"
 description = "System crate for the Mozilla SpiderMonkey JavaScript engine."
 repository.workspace = true
-version = "0.140.0-2"
+version = "0.140.0-3"
 authors = ["Mozilla"]
 links = "mozjs"
 license.workspace = true
 exclude = [
-  "mozjs/js/src/tests/**",
-  "mozjs/js/src/octane/**",
-  "mozjs/js/src/jit-test/**",
-  "mozjs/js/src/jsapi-tests/**",
-  "mozjs/js/src/doc/**",
+  "__pycache__",
+  "/etc/",
+  "/mozjs/.cargo/",
+  "/mozjs/python/mozperftest/mozperftest/system/example.zip",
+  "/mozjs/python/mozperftest/mozperftest/tests/data",
+  "/mozjs/third_party/python/aiohttp",
+  # pip is 8MiB and we can assume its installed.
+  "/mozjs/third_party/python/pip",
+  "/mozjs/testing/mozharness",
+  "/mozjs/testing/mozbase/mozproxy",
+  "/mozjs/testing/mozbase/**/tests",
+  "/mozjs/testing/performance/",
+  "/mozjs/testing/test/",
```

Comment:
Why not add these to filters: https://github.com/servo/mozjs/blob/main/mozjs-sys/etc/filters.txt

Reply:
I was thinking it might make it easier to tweak the excludes if we can easily inspect the contents of excluded folders locally. 
I'm not particulary opposed though - If you prefer, then we can use filters.txt instead.

Reply:
The idea is to make repository (checkout) more lightweight (thus also cargo checkout when using git) and diffs of SM bumps smaller. If anybody wants to inspect content of filtered files it can read SM archive. I also think it makes sense to have all SM filtering/excluding in one place.

Reply:
Does cargo checkout repositories using a shallow clone? I guess it does make sense to at least try to prevent the repository bloating further than it already has, but this won't reduce the size of standard checkouts, since everything is still in the history.